robot/packages/cloud_package_manager_test.go

```diff
@@ -424,6 +445,7 @@ func TestPackageRefs(t *testing.T) {
 	test.That(t, err, test.ShouldBeNil)
 
 	t.Run("PackagePath", func(t *testing.T) {
+		// FYI: This test passes in CI but fails when ran on MacOS
```

Comment:
imo worth adding something like
```go
if runtime.GOOS == "darwin" {t.Skip("skip on macos because ...")}
```

is the failure a runtime issue as well on macs? or just tests